core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -636,9 +637,10 @@ describe('Translation Modal Component', () => {
 
     describe('when caption is not changed in copied images', () => {
       it('should not submit the translation', () => {
-        component.textToTranslate = '<oppia-noninteractive-image alt-with-' +
+        component.textToTranslate = '<oppia' +
+          '-noninteractive-image alt-with-' +
           'value="&amp;quot;Image description&amp;quot;" caption-with-value=' +
-          '"&amp;quot;Image caption&amp;quot;" filepath-with-value="&amp;quot' +
+          '"&amp;quot;Image text&amp;quot;" filepath-with-value="&amp;quot' +
```

Comment:
Why are these changes necessary? The test case is `when caption is not changed`, so I would expect the image caption to be the same and not change.